slider.js

```diff
@@ -0,0 +1,26 @@
+//Created by Cole Hemp(GamePlayerCole)
```

Comment:
This file doesn't seem to add anything to our project. If it does, please point out the functionality in a comment below the other comments.